roles/remove_cloudnativepg_sandbox/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+# tasks file for RemoveCloudNativePG-Sandbox
+- name: Remove 'cnpg-sandbox-monitored-cluster.yaml' manifest from k8s cluster
+  kubernetes.core.k8s:
+    state: absent
+    namespace: default
+    src: ./cnpg-sandbox-monitored-cluster.yml
```

Comment:
Once the content of `cnpg-sandbox-monitored-cluster.yml` has been converted into variables, we can use here something like:
```yaml
- name: Remove CNP manifest from k8s cluster
  kubernetes.core.k8s:
    state: absent
    namespace: default
    definition:
      apiVersion: "{{ cnp_api_version }}"
      kind: Cluster
      metadata:
        name: "{{ cnp_cluster_name }}"
      spec:
        instances: "{{ cnp_n_instances }}"
        storage:
          size: "{{ cnp_storage_isize }}"
        monitoring:
          enablePodMonitor: "{{ cnp_enable_pod_monitoring | bool }}"
```

Reply:
This is not possible because the k8s engine does not unmarshall the variables in ansible in go